handbook/business-operations/people-operations.md

```diff
@@ -143,308 +143,36 @@ At the end of their first two weeks of onboarding at Fleet, every new team membe
 
```

Comment:
"Moved tracking hours, hiring, departures to leadership"

(Better commit message)